.github/workflows/publish-packages-1.0.yml

```
@@ -49,7 +43,7 @@ jobs:
     - name: Publish Artifacts
       uses: actions/upload-artifact@v3
       with:
-        name: ${{ matrix.os }}-packages
+        name: windows-latest-packages
```

Meh? I mean I think this is the name that was being used before, but the prefix `windows-latest` doesn't really mean anything here. Probably can just be 

```suggestion
        name: published-packages
```

or something.